include/contraction/contractionGraph.hpp

```diff
@@ -151,74 +200,97 @@ class Pgr_contractionGraph : public Pgr_base_graph<G, CH_vertex, CH_edge, t_dire
         return os;
     }
 
-
-    /*! @brief add_shortuct to the graph during contraction
-
-    @param [in] edge of type *CH_edge* is to be added
-    @param [in] u vertex
-    @param [in] v vertex
-
-    u -> w -> v
-
-    u -> v
-
-    edge (u, v) is a new edge e
-    contracted_vertices = w + contracted vertices
+    /*!
+        @brief add a shortcut to the graph during contraction
+        @param [in] edge of type *CH_edge* is to be added
+        @param [in] u vertex
+        @param [in] v vertex
+        u -> w -> v
+        u -> v
+        edge (u, v) is a new edge e
+        contracted_vertices = w + contracted vertices
     */
-
     void add_shortcut(const CH_edge &edge, V u, V v) {
         bool inserted;
         E e;
         if (edge.cost < 0) return;
-
         boost::tie(e, inserted) = boost::add_edge(u, v, this->graph);
-
         this->graph[e]= edge;
     }
 
+    /*
+     *
+     * u ----e1{v1}----> v ----e2{v2}----> w
+     *
+     * e1: min cost edge from u to v
+     * e2: min cost edge from v to w
+     *
+     * result:
+     * u ---{v+v1+v2}---> w
```

Comment:
You're notation is not clear. We get `e1{v1}` and `e2{v2}`, then move towards `{v+v1+v2}`. What are `v`, `v1`, `v2`? Vertices? Plus, the existence of a `v` vertex in the original situation makes this last notation confusing.

Reply:
Ok, that's an existing piece of docstring. Let's forget my comment.